src/pymovements/plotting/main_sequence_plot.py

```diff
@@ -21,21 +21,24 @@
 from __future__ import annotations
 
 import matplotlib.pyplot as plt
+import numpy as np
 import polars as pl
 from matplotlib.collections import Collection
+from sklearn.metrics import r2_score
 
 from pymovements.events import EventDataFrame
 
 
 def main_sequence_plot(
```

Comment:
add one argument for turning the fit and the R2 on and off.

something like this:
```python
fit: bool = True,
measure: bool = True,
```

Although we could also call it line, we should keep in mind that we might want to do non-linear fits somewhen.
Having the more general name would enable us to use the argument to specify the method in the future like `fit='linear'` or `fit='exponential'`, where `fit=True` would be equivalent to `fit='linear'`.

The same holds true for the R2 measure. In contrast to what I said yesterday, R2 is not a valid measure for nonlinear fits, so here too, we could specify measures in the future, e.g. standard error of regression S.

These two blog posts are probably good inspiration for making this PR future proof for follow ups:
https://blog.minitab.com/en/adventures-in-statistics-2/why-is-there-no-r-squared-for-nonlinear-regression
https://blog.minitab.com/en/adventures-in-statistics-2/regression-analysis-how-to-interpret-s-the-standard-error-of-the-regression